examples/org.eclipse.swt.examples/src/org/eclipse/swt/examples/graphics/TransformReuseTab.java

```diff
@@ -0,0 +1,119 @@
+/*******************************************************************************
+ * Copyright (c) 2025 Yatta.
```

Comment:
Maybe align with other copyright notices?
```suggestion
 * Copyright (c) 2025 Yatta and others.
```

Reply:
Sure